src/SNLS_TrDLDenseG.h

```diff
@@ -84,6 +84,7 @@ class SNLSTrDlDenseG
       __snls_hdev__ double  getRhoLast() const { return(_rhoLast); };
       __snls_hdev__ double  getDelta  () const { return(_delta  ); };
       __snls_hdev__ double  getRes    () const { return(_res    ); };
+      __snls_hdev__ SNLSStatus_t getStatus() const { return(_status); };
```

Comment:
Parentheses aren't needed for C++ in the return statement.  It could just read `return _status;`